http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -452,6 +441,27 @@ private static void setKeepAlive(SelectableChannel selectable, Duration tcpKeepA
         }
     }
 
+    private static HttpClientTransport getClientTransport(ClientConnector connector, HttpClientConfig config)
+    {
+        ImmutableList.Builder<ClientConnectionFactory.Info> protocols = ImmutableList.builder();
+        if (config.isHttp2Enabled()) {
+            HTTP2Client http2Client = new HTTP2Client(connector);
+            http2Client.setUseALPN(true);
```

Comment:
This is default, so we shouldn't need to set it

Reply:
Removed.